pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,564 @@
+    echo "${OPTION}=no-threads" >> ${OUTPUT_FILE}
+    echo "${OPTION}=one-thread-per-connection" >> ${OUTPUT_FILE}
+    echo "${OPTION}=dynamically-loaded" >> ${OUTPUT_FILE}
+    echo "${OPTION}=pool-of-threads" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--thread-pool-high-prio-mode" ]; then
+    echoit "  > Adding possible values transactions, statements for option '${OPTION}' to the final list..."
+    echo "${OPTION}=transactions" >> ${OUTPUT_FILE}
+    echo "${OPTION}=statements" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--transaction-isolation" ]; then
+    echoit "  > Adding possible values READ-UNCOMMITTED, READ-COMMITTED for option '${OPTION}' to the final list..."
+    echo "${OPTION}=READ-UNCOMMITTED" >> ${OUTPUT_FILE}
+    echo "${OPTION}=READ-COMMITTED" >> ${OUTPUT_FILE}
+    echo "${OPTION}=REPEATABLE-READ" >> ${OUTPUT_FILE}
+    echo "${OPTION}=SERIALIZABLE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--utility-user-schema-access" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--utility-user-privileges" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--proxy-protocol-networks" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--disabled-storage-engines" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--innodb-temp-data-file-path" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--innodb-undo-directory" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--log-syslog-tag" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--innodb-doublewrite-file" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--plugin-load" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--sql-mode" ]; then
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--innodb-monitor-gaplock-query-filename" ]; then                  ## fb-mysql
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--innodb-tmpdir" ]; then                                          ## fb-mysql
+    echoit "  > Adding possible values ... for option '${OPTION}' to the final list..."
+  elif [ "${OPTION}" == "--rocksdb-compact-cf" ]; then                                     ## fb-mysql
```

Comment:
Remove all rocksdb options from the script. This is redundant code since rocksdb variables are not available in mysqld help by default

Reply:
Fixed : Removed RocksDB options